ssh.go

```diff
@@ -72,9 +72,9 @@ type Window struct {
 
 // Pty represents a PTY request and configuration.
 type Pty struct {
-	Term   string
-	Window Window
-	// HELP WANTED: terminal modes!
+	Term      string
```

Comment:
This looks like it needs a gofmt

Reply:
Oh sorry, I forgot it. I will modify this.